frontend/src/views/main/project/TestSuites.vue

```diff
@@ -48,9 +48,17 @@ onActivated(async () => await testSuitesStore.loadTestSuites(props.projectId))
 
 async function createTestSuite() {
     const project = await api.getProject(props.projectId)
+    const date = new Date();
+    const formattedDate = date.getFullYear() + '-' +
```

Comment:
we should use `formatDate` inside of  `filters`. Plus there is better way to format by using format template `YYYY-MM-DD HH:mm:ss`

Reply:
Actually, I'm gonna remove this date from the name, because the user will be able to change the suite name from Testing tab.